styles.css

```diff
@@ -302,3 +306,102 @@ body {
   margin-left: 16px;
   font-size: 1em;
 }
+
+.contact {
+  border-top-left-radius: 105px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 30px;
+  background: #6070ff url(images/contact-form-shapes-mobile.png) no-repeat center;
+  background-size: 100% 90%;
+  margin-top: 25%;
+}
```

Comment:
- [ ] Kindly consider adding a `background-image` property to the element in order to match the background image specified in the `Figma` design. This will help ensure visual consistency between the design and the implemented code. Please make sure to use the appropriate CSS syntax and provide the correct path or URL for the background image to achieve the desired result. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="311" alt="Screenshot 2023-07-11 at 19 45 49" src="https://github.com/yayaismaail/my-portfolio/assets/107192076/b1f76706-ef5a-4ff4-849e-45d24fdb9958">
</td>
<td>
<img width="209" alt="Screenshot 2023-07-11 at 19 37 44" src="https://github.com/yayaismaail/my-portfolio/assets/107192076/e8f43ae6-e1c8-4664-a7d9-87dbabcb04d9">
</td>
</tr>
</table>